python_scripts/01_tabular_data_exploration.py

```diff
@@ -233,9 +233,9 @@
 #   is a bit lighter that for the left region which means that the algorithm is
 #   not as certain in this region.
 # * In the region `age > 28.5 AND hours-per-week > 40.5` (top-right region),
-#   the prediction is `low-income`. The probability of the class `low-income`
-#   is very close to 0.5 which means the model is not sure at all about its
-#   prediction.
+#   the prediction is `low-income`. However the probability of the class
```

Comment:
this one is not a typo but I find it a bit more clear to emphasize that the prediction can't be trusted